### DISCUSSION TITLE
Visual Studio 2022 for Mac - iOS Simulator

### DISCUSSION THREAD

**Discussion by nspyrou** [CommunityToolkit/Maui#1095] [General]
I have a strange behavior regarding Visual Studio for Mac and Maui Community Toolkit (v 5.0). 

I 've setup a blank MAUI project to see whether this is happening in general or not. Two blank pages with a Label, using Shell.Current.GotoAsync() for navigation everything works well. In the same project I installed Community Toolkit from NuGet, Initialized in the MauiProgram.cs - Set the Routes - Setup a ViewModel did a couple of test bindings and setup a couple of ICommand Observable properties for the Views and run it again. I got a Microsoft.Maui.Controls Exception after I navigated to another rather than the Main Page ... 

I tried to see if this is a random thing or it is done in general. So I tranferred my project from the Windows machine to the Mac and did the same thing. Attached the error I get with the stacktrace. 

Anybody know why this is happening?

![image](https://user-images.githubusercontent.com/8970768/225095732-0050ec8f-e840-471f-af4c-2834f6b26862.png)

The Mac is a MacMini M2 with MacOS Ventura and XCode 14.2.

The program breaks inside the Platforms\iOS\Program.cs:

![image](https://user-images.githubusercontent.com/8970768/225096934-f0be4987-13bc-43f7-8cd0-df27ef82f366.png)

Thanks.

**Comment by VladislavAntonyuk** [comment]
Please create an issue and attach the repro
